hw/ip/aes/aes.core

```diff
@@ -24,8 +24,58 @@ filesets:
       - rtl/aes.sv
     file_type: systemVerilogSource
 
+  files_verilator_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+    files:
+      - lint/aes.vlt
+    file_type: vlt
+
+  files_ascentlint_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+      # tool setup
+      - lowrisc:lint:ascentlint
+    files:
+      - lint/aes.waiver
+    file_type: waiver
+
+
+parameters:
+  SYNTHESIS:
+    datatype: bool
+    paramtype: vlogdefine
+
+
 targets:
   default: &default_target
     filesets:
+      # Note on Verilator waivers:
+      # You *must* include the waiver file first, otherwise only global waivers
+      # are applied, but not file-specific waivers.
```

Comment:
That's actually not the case any more, fusesoc now takes care to always include the vlt files first.

Reply:
Ah ok thanks for clarifying. I basically just preserved a comment present in the ibex repository here.

Reply:
removed